### PR TITLE
Support for Bitbucket Pipelines

### LINKED ISSUE #2: Bitbucket Pipelines
https://support.atlassian.com/bitbucket-cloud/docs/write-a-pipe-for-bitbucket-pipelines/

### PR DESCRIPTION
**Pull request by keith-hall**
Fixes #2

### CODE REVIEW

**Comment by michaelblyons** [comment]
Rename to "Bitbucket Pipeline" ? For whatever reason, they don't capitalize the second B. 

There's something else called "Bitbucket Pipe" that also exists. I haven't used one of those, though. It seems to have metadata.

File: BitBucket.sublime-syntax [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+extends: Packages/YamlPipelines/YamlPipeline.sublime-syntax
+
+file_extensions:
+  - bitbucket-pipelines.yml
+
+contexts:
+  node:
+    - meta_prepend: true
+    - match: \s(script)\s*(:)(?=\s|$)
```
Comment (michaelblyons) [inline BitBucket.sublime-syntax]: Start with `^\s+` instead?

File: BitBucket.sublime-syntax [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      captures:
+        1: string.unquoted.plain.out.yaml keyword.control.flow.script.pipeline
+        2: punctuation.separator.key-value.yaml
+      push: script-block-sequence
+
+  script-block-sequence:
+    - match: (-)(?!\S)
```
Comment (michaelblyons) [inline BitBucket.sublime-syntax]: How about `(-)[ \t]+(?=\S)`?

**Review by michaelblyons — CHANGES_REQUESTED** [review]

File: BitBucket.sublime-syntax (hunk 2 above)
Comment (michaelblyons) [reply BitBucket.sublime-syntax]: And change `scope` below to `capture` number 1.

Comment (keith-hall) [reply BitBucket.sublime-syntax]: done :)

File: BitBucket.sublime-syntax (hunk 1 above)
Comment (keith-hall) [reply BitBucket.sublime-syntax]: done :)

**Review by michaelblyons — APPROVED** [review]

**Pull request merged**